src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -229,7 +226,7 @@ private void assertAllowedMimeType(final String mimeType) {
 
   private void assertAllowedClassification(final String classificationKey) {
     final Map<String, String> allowedClassifications = getAllowedClassifications();
-    if (!allowedClassifications.containsKey(classificationKey)) {
+    if (!allowedClassifications.containsKey(classificationKey) && classificationKey != null) {
```

Comment:
This line should be changed to `if (classificationKey != null && !allowedClassifications.containsKey(classificationKey) )`